app/src/main/java/com/kickstarter/viewmodels/ProjectCardHolderViewModel.java

```diff
@@ -0,0 +1,279 @@
+    Observable<String> photoUrl();
+    Observable<String> blurbText();
+    Observable<String> nameText();
+    Observable<Project> notifyDelegateOfProjectClick();
+    Observable<Boolean> potdViewGroupIsGone();
+    Observable<DateTime> projectCanceledAt();
+    Observable<DateTime> projectFailedAt();
+    Observable<Boolean> projectStateViewGroupIsGone();
+    Observable<DateTime> projectSuccessfulAt();
+    Observable<DateTime> projectSuspendedAt();
+    Observable<String> rootCategoryName();
+    Observable<Void> setDefaultTopPadding();
+    Observable<Void> setMetadataTopPadding();
+    Observable<Boolean> starredViewGroupIsGone();
+    Observable<Boolean> successfullyFundedTextViewIsGone();
+  }
+
+  final class ViewModel extends ActivityViewModel<ProjectCardViewHolder> implements Inputs, Outputs {
+
+    public ViewModel(final @NonNull Environment environment) {
+      super(environment);
+
+      final Observable<Boolean> projectInEndedStates = this.project
+        .map(
+          p -> p.state().equals(Project.STATE_SUCCESSFUL)
+            || p.state().equals(Project.STATE_CANCELED)
+            || p.state().equals(Project.STATE_FAILED)
+            || p.state().equals(Project.STATE_SUSPENDED)
+        );
+
+      this.projectOutput = this.project;
+
+      this.backersCountText = this.project
+        .map(Project::backersCount)
+        .map(NumberUtils::format);
+
+      this.backingViewGroupIsGone = this.project
+        .map(p -> p.isBacking() || p.isStarred() || p.isPotdToday() || p.isFeaturedToday());
+
+      this.blurbText = this.project
+        .map(Project::blurb);
+
+      this.categoryNameText = this.project
+        .map(Project::category)
+        .map(Category::name)
+        .compose(coalesce(""));
+
+      this.deadlineCountdownText = this.project
+        .map(ProjectUtils::deadlineCountdownValue)
+        .map(NumberUtils::format);
+
+      this.featuredViewGroupIsGone = this.project
+        .map(p -> p.isBacking() || p.isStarred() || p.isPotdToday() || !p.isFeaturedToday());
+
+      this.friendAvatarUrl = this.project
+        .filter(Project::isFriendBacking)
+        .map(Project::friends)
+        .map(friends -> friends.get(0).avatar().small());
+
+      this.friendBackingViewIsHidden = this.project
+        .map(Project::isFriendBacking)
+        .map(BooleanUtils::negate);
+
+      this.friendsForNamepile = this.project
+        .filter(Project::isFriendBacking)
+        .map(Project::friends);
+
+      this.fundingUnsuccessfulTextViewIsGone = this.project
+        .map(p ->
+          !p.state().equals(Project.STATE_CANCELED) &&
+            !p.state().equals(Project.STATE_FAILED) &&
+            !p.state().equals(Project.STATE_SUSPENDED));
+
+      this.imageIsInvisible = this.project
+        .map(Project::photo)
+        .map(ObjectUtils::isNull);
+
+      this.metadataViewGroupIsGone = this.project
+        .map(p -> !p.isBacking() && !p.isStarred() && !p.isPotdToday() && !p.isFeaturedToday());
```

Comment:
looks like this observable is precisely `this.backingViewGroupIsGone.map(isFalse)`. any benefits from thinking of it as such?